src/OpenTelemetry.Instrumentation.Hangfire/TracerProviderBuilderExtensions.cs

```diff
@@ -33,7 +34,25 @@ public static TracerProviderBuilder AddHangfireInstrumentation(this TracerProvid
     {
         Guard.ThrowIfNull(builder);
 
-        Hangfire.GlobalJobFilters.Filters.Add(new HangfireInstrumentationJobFilterAttribute());
+        return builder.AddHangfireInstrumentation(null);
+    }
+
+    /// <summary>
+    /// Adds Hangfire instrumentation to the tracer provider.
+    /// </summary>
+    /// <param name="builder"><see cref="TracerProviderBuilder"/> being configured.</param>
+    /// <param name="configureHangfireInstrumentationOptions">Callback action for configuring <see cref="HangfireInstrumentationOptions"/>.</param>
+    /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
+    public static TracerProviderBuilder AddHangfireInstrumentation(
+        this TracerProviderBuilder builder,
+        Action<HangfireInstrumentationOptions> configureHangfireInstrumentationOptions)
```

Comment:
Most of the extensions methods follow contract with configuration option as optional instead of having two specific methods. 

```csharp
    public static TracerProviderBuilder AddHangfireInstrumentation(
        this TracerProviderBuilder builder,
        Action<HangfireInstrumentationOptions> configureHangfireInstrumentationOptions = null)
```

All Instrumentation.Hangfire releases are marked as beta so we can do it here also.

Reply:
"All Instrumentation.Hangfire releases are marked as beta so we can do it here also."

@Kielek  how to do that?

Reply:
@vitor-pinto-maersk, I mean that you can introduce breaking change to the API, as there were no stable release.

LGTM